src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_DocumentHighlights.cs

```diff
@@ -16,7 +16,8 @@ internal partial class CodeAnalysisService : IRemoteDocumentHighlights
         {
             var solution = await GetSolutionAsync().ConfigureAwait(false);
             var document = solution.GetDocument(documentId);
-            var documentsToSearch = ImmutableHashSet.CreateRange(documentIdsToSearch.Select(solution.GetDocument));
+            var documentsToSearch = ImmutableHashSet.CreateRange(
```

Comment:
removing this to let things crash might make bug more explicit. with this, we might not crash and didn't know that some document just silent failed to run HR. same for FAR.